Adding the feedbackCard component

NathOrtega opened the pull request.

NathOrtega committed:
Adding the feedbackCard component
making some needed changes to multiple components

NathOrtega committed:
Making some changes to improve components

Jeangel commented on src/components/misc/FeedbackCard/index.tsx:
```diff
@@ -0,0 +1,136 @@
+      h={[200, 151, 151]} 
+      borderRadius='10px' p='24px' 
+      justifyContent='space-between'
+      sx={{
```
Let's use here [Chakra responsive styles](https://chakra-ui.com/docs/styled-system/features/responsive-styles#the-object-syntax). I think the object syntax would be more appropriate.
That also applies to other places where you used media-queries

Jeangel commented on src/components/misc/FeedbackCard/index.tsx:
```diff
@@ -0,0 +1,136 @@
+        </Text>
+        <Badge variant='feedbackTag'>{category}</Badge>
+      </Flex>
+      <Badge 
```
According to the designs, we should use the `Toggle` component here.
The behavior should be that once toggled, it will add a +1 to the total votes count.

Jeangel commented on src/components/misc/FeedbackCard/index.tsx:
```diff
@@ -0,0 +1,136 @@
+        <Icon 
+          as={FaComment} 
+          marginRight='4px' 
+          color='#CDD2EE'
```
Let's see if there is a color in our palette that looks similar to this color.

NathOrtega committed:
Merge branch 'main' into feature/feedback-card

# Conflicts:
#	src/components/misc/RoadmapCard/index.tsx

NathOrtega committed:
Addresing feedback 🛠

- Using chakra responsive styles instead of media queries.
- Replacing the Badge component for the Toggle component.
- Changing Comments Icon color to use our palette.

Jeangel committed:
Formatting code 📐
